core/tx_pool.go

```diff
@@ -231,11 +232,12 @@ type TxPool struct {
 	locals  *accountSet // Set of local transaction to exempt from eviction rules
 	journal *txJournal  // Journal of local transaction to back up to disk
 
-	pending map[common.Address]*txList   // All currently processable transactions
-	queue   map[common.Address]*txList   // Queued but non-processable transactions
-	beats   map[common.Address]time.Time // Last heartbeat from each known account
-	all     *txLookup                    // All transactions to allow lookups
-	priced  *txPricedList                // All transactions sorted by price
+	pending   map[common.Address]*txList   // All currently processable transactions
+	queue     map[common.Address]*txList   // Queued but non-processable transactions
+	beats     map[common.Address]time.Time // Last heartbeat from each known account
+	queued_ts map[common.Hash]time.Time    // Timestamp for when queued transactions were added
```

Comment:
Nitpick: it's frowned upon in golang to use underscore -- go uses camelCase instead. Please change this